variable_distance_exploration/lib/experimental_media_query.dart

```diff
@@ -67,21 +67,23 @@ class ExperimentalMediaQuery extends MediaQueryData {
   /// Default is [DeviceCorporealType.slabHandheld].
   final DeviceCorporealType deviceCorporealType;
 
+  /// If the phone is foldable, is it folded (true) or not (false).
+  ///
+  /// If the phone is not foldable, null.
+  ///
+  /// Default is null.
+  final bool isFolded;
+
   /// Number of users facing the screen.
   ///
   /// Null means 'unknown'.
   ///
   /// Default is null.
   final int viewerCount;
 
-  /// If the phone is foldable, is it folded (true) or not (false).
-  /// 
-  /// If the phone is not foldable, null.
+  /// Altitude angle of eyes of the main user to the screen. The horizon is the
```

Comment:
Sorry to keep nitting on this, but the first sentence should be it's own paragraph.

https://www.dartlang.org/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph